packages/frontend/src/main.tsx

```diff
@@ -33,8 +33,11 @@ const router = createBrowserRouter([
             const courses = (await fetch(BASE_URL).then((res) =>
```

Comment:
You can do these in paralell using [Promise.all](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all)